test_context.go

```diff
@@ -331,20 +330,4 @@ func (ctx *ScenarioContext) stepWithKeyword(expr interface{}, stepFunc interface
 	ctx.suite.steps = append(ctx.suite.steps, def)
 }
 
-// Build creates a test package like go test command at given target path.
-// If there are no go files in tested directory, then
-// it simply builds a godog executable to scan features.
-//
-// If there are go test files, it first builds a test
-// package with standard go test command.
-//
-// Finally it generates godog suite executable which
-// registers exported godog contexts from the test files
-// of tested package.
-//
-// Returns the path to generated executable
-func Build(bin string) error {
```

Comment:
Removing this function would be a breaking change, let's deprecate it instead.

Reply:
when add `// Deprecated: use xxxx instead` we should tell user which function to instead ?

Reply:
You can use something free formed like `// Deprecated: feature will be removed, no substitute`. The message comes from your reasoning why you wanted to remove that function in the first place.

It's ok to keep it as is for now.